client/src/components/CFP/CFPForm.js

```diff
@@ -119,8 +125,9 @@ class CFPForm extends Component {
             </Button>
           </div>
         </form>
+
+        {/* <MultiStep showNavigation={true} steps={steps} user={user} /> */}
```

Comment:
I commented out this line of code but it shows the way I thought of passing the prop `user` to the parent component `MultiStep`. This prop was supposed to be used by the first step of the form where the user is asked to fill their personal details.
You can see below in `StepOne.js` the part of the form I wanted to use it in.